### PR TITLE
[INLONG-9060][Manager] Fix manager return wrong sink configuration to sort standalone

### LINKED ISSUE #9060: [Bug][Manager] Manager return wrong sink configuration to sort standalone
### What happened

The sink have six field :  id name source sequence remark date 

But when sort-standalone request  manager for this sink config, and Manager return wrong sink configuration
image

### What you expected to happen

Manager return correct field information such as :
image

### How to reproduce

when sort-standalone request manager for sink config

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by castorqin**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #9060 

### Motivation

1. update StreamSinkFieldEntityMapper.selectAllFields method, use table inlong_stream_field replace stream_sink_field table
2. if stream use extended fields , add FILED_OFFSET params 

### Modifications
The details can be found at [#9060 ](https://github.com/apache/inlong/issues/9060).

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/pulsar/PulsarSinkOperator.java [hunk 1]
```diff
@@ -123,7 +123,8 @@ public Map<String, String> parse2IdParams(StreamSinkEntity streamSink, List<Stri
     }
 
     private String getFullTopicName(PulsarSinkDTO pulsarSinkDTO) {
-        return pulsarSinkDTO.getPulsarTenant() + "/" + pulsarSinkDTO.getNamespace() + "/" + pulsarSinkDTO.getTopic();
+        return "persistent://" + pulsarSinkDTO.getPulsarTenant() + "/" + pulsarSinkDTO.getNamespace() + "/"
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/pulsar/PulsarSinkOperator.java:126]: Use `PULSAR_TOPIC_FORMAT`

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java [hunk 2]
```diff
@@ -278,6 +292,18 @@ private List<Map<String, String>> parseIdParams(List<StreamSinkEntity> streams,
                 .collect(Collectors.toList());
     }
 
+    private Map<String, String> setFiledOffset(StreamSinkEntity streamSink, Map<String, String> params) {
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java:295]: void setFiledOffset(StreamSinkEntity streamSink, Map<String, String> params)

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java [hunk 3]
```diff
@@ -278,6 +292,18 @@ private List<Map<String, String>> parseIdParams(List<StreamSinkEntity> streams,
...
+
+        SortSourceStreamInfo sortSourceStreamInfo = allStreams.get(streamSink.getInlongGroupId())
+                .get(streamSink.getInlongStreamId());
+        InlongStreamExtParam inlongStreamExtParam = JsonUtils.parseObject(
+                sortSourceStreamInfo.getExtParams(), InlongStreamExtParam.class);
+        if (inlongStreamExtParam != null && !inlongStreamExtParam.getUseExtendedFields()) {
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java:301]: CollectionUtils.isNotEmpty()

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/pulsar/PulsarSinkOperator.java (hunk 1 above)
Comment (castorqin) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/pulsar/PulsarSinkOperator.java]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java (hunk 2 above)
Comment (castorqin) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java (hunk 3 above)
Comment (castorqin) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/cls/ClsOperator.java [hunk 4]
```diff
@@ -55,26 +56,28 @@
 @Service
 public class ClsOperator {
 
+    @Value("${cls.manager.endpoint}")
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/cls/ClsOperator.java:59]: Please add the configuration to `application-dev.properties` and add relevant comments.

Comment (castorqin) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/cls/ClsOperator.java:59]: done

**Review by fuweng11 — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**